build.gradle

```diff
@@ -482,7 +482,10 @@ subprojects {
 
     exclude testsToExclude
 
-    useJUnitPlatform()
+    useJUnitPlatform {
+      includeTags "integration"
```

Comment:
this is incorrect. we have three test-related tasks: `test`, `unitTest` and `integrationTest`. `test` does not need to include/exclude tags. `unitTest` needs `excludeTags "integration"` and `integrationTest` needs `includeTags "integration"`